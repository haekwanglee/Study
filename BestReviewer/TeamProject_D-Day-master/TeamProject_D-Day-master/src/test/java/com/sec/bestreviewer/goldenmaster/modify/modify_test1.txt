MOD, , , ,certi,ADV,name,Modified Pro
SCH,-p, , ,certi,ADV