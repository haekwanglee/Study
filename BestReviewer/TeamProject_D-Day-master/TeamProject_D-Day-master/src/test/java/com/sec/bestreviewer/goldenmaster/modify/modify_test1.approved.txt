MOD,38
SCH,90000000,Modified Pro,CL2,010-1111-1111,19980101,ADV
SCH,90000001,Modified Pro,CL2,010-1111-1112,19980102,ADV
SCH,91000000,Modified Pro,CL2,010-1111-1113,19980103,ADV
SCH,91000001,Modified Pro,CL2,010-1111-1114,19980104,ADV
SCH,92000000,Modified Pro,CL2,010-1111-1115,19980105,ADV
